ez_setup.py

```diff
@@ -54,7 +54,9 @@ def _python_cmd(*args):
 
```

Comment:
this can go, why is this in the template :nauseated_face:

Reply:
It wasn't deleted?